docs/GettingStarted.md

```diff
@@ -119,6 +108,18 @@ You are now set up to use all ES6 features and React specific syntax.
 }
 ```
 
+#### Babel 6
+
+Jest 24 dropped support for Babel 6. We highly recommend you to upgrade to Babel 7, which is actively maintained. If you for some reason cannot upgrade, you can keep using Jest 23 or you can upgrade to Jest 24 but keep `babel-jest` at version 23, which will continue to work for now:
```

Comment:
```suggestion
Jest 24 dropped support for Babel 6. We highly recommend you to upgrade to Babel 7, which is actively maintained. However, if you cannot upgrade to Babel 7, either keep using Jest 23 or upgrade to Jest 24 with `babel-jest` locked at version 23, like in example below:
```

Reply:
About the last part: I kinda felt like "for now" was good to emphasize that this is not really supported and might break in a Jest 24 minor, don't you think?

Reply:
Maybe like this in another sentence? (Can you take a look again please)